content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,74 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect a Helpdesk ticket to a Project task. While working on the ticket, use either the ticket or 
+the task to record timesheets. When the ticket is solved, Odoo can pull from those timesheets to 
+bill the client. This integration provides more control and transparency over what the client is 
+charged for.
 
 Configuration
 =============
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`. Select a Helpdesk team and 
+click :guilabel:`Edit`. Then, enable the options :guilabel:`Timesheet on Ticket` and 
+:guilabel:`Time Reinvoicing`.
```

Comment:
Small suggestion to improve readability. Might just be style difference, however to me this small change helps connect the flow between different actions in the same breath, as opposed to brief instructions that may or may not appear related.

```suggestion
Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, select a Helpdesk team, and
click :guilabel:`Edit`. Then, enable the options :guilabel:`Timesheet on Ticket` and
:guilabel:`Time Reinvoicing`.
```